packages/vscode-extension/src/extension.ts

```diff
@@ -91,7 +91,9 @@ export async function activate(context: vscode.ExtensionContext) {
       m365TokenProvider: M365TokenInstance,
     });
 
-    registerCodelensAndHoverProviders(context);
+    if (vscode.workspace.isTrusted) {
+      registerCodelensAndHoverProviders(context);
```

Comment:
How about other operations like debugging and tasks?

Reply:
They are all blocked by VS Code.